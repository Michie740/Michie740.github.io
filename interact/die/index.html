<!DOCTYPE html>
<html>
    <meta charset="utf-8">
    <meta name = "viewport" content = "width=device-width, initial-scale=1.0">
    <head>
        <title>
            Live Like You Will Die Today
        </title>
        <link rel="stylesheet" href="../midtermwireframe/wfstyle.css">

    </head>
     
    <body id = "die">
        <header class = "diepage">
            <h1>Fight</h1>
        </header>
        <nav class = "navigation">
            <ul>
                <li>
                    <a href = "../../">Home</a>
                </li>
                <li>
                    <a href = "../midtermwireframe/wireframe.html">More Poetry</a>
                </li>
            </ul>
        </nav>
        <div class = "noreg">
            <div class = "poem">
                So many gasps, but no way to breathe. <br>
                Go under, distress, it’s a losing fight. <br>
                Scream, useless, only overwhelming silence. <br>
                <br>
                The waters rise, it’s not an easy fight <br>
                when pressure and stress take advantage of the silence <br>
                and thoughts and voices get too loud, and it gets harder to breathe. <br>
                <br>
                Suppress the thoughts, suppress the silence. <br>
                It’ll get better - It’s okay, just breathe. <br>
                Don't get swept away, don’t give up the fight. <br>
                
                Take a deep breath - hold - then fight to escape, to shatter the silence, to breathe. <br>

            </div>
        </div>
        <footer>
            <h4>
                Michelle Yang 2020
            </h4>
        </footer>
    </body>
</html>