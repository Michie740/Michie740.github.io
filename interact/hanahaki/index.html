<!DOCTYPE html>
<html>
    <meta charset="utf-8">
    <meta name = "viewport" content = "width=device-width, initial-scale=1.0">
    <head>
        <title>
            how could something so beautiful be so cruel
        </title>
        <link rel="stylesheet" href="../midtermwireframe/wfstyle.css">

    </head>
     
    <body class = "hanahakipage">
        <header id = "hanahaki">
            <h1>how could something so beautiful be so cruel</h1>
        </header>
        <nav class = "navigation">
            <ul>
                <li>
                    <a href = "../../">Home</a>
                </li>
                <li>
                    <a href = "../midtermwireframe/wireframe.html">More Poetry</a>
                </li>
            </ul>
        </nav>
        <div class = "flowers">
            <div class = "poem">
                <img src = "https://pbs.twimg.com/media/CfG8epuUYAEfhKV.jpg" height = "150px">

                <!-- Cherry blossoms bloomed <br>
                in the garden I walked through <br>
                where I first met you <br>
                <br>
                we hung out all spring <br>
                more enamoured day by day <br>
                did you feel the same? <br>
                <br>
                summer months snuck up <br>
                that’s when the first petal came <br>
                daisy - your favorite <br>
                <br>
                the days passed by quick <br>
                no secrets between us but <br>
                this I couldn’t share <br>
                <br>
                summer is ending <br>
                I think soon I will too as <br>
                my lungs tighten up <br>
                <br>
                I cough and look down <br>
                and see blossoms pretty as <br>
                when I first met you <br>
                <br>
                I’m in love, I know <br>
                but you never will so like <br>
                leaves in fall, I’ll fall. <br>
                <br> -->
            </div>
        </div>
        <footer>
            <h4>
                Michelle Yang 2020
            </h4>
        </footer>
    </body>
</html>