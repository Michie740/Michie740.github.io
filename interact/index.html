<!DOCTYPE html>
<html>
    <meta charset="utf-8">
    <meta name = "viewport" content = "width=device-width, initial-scale=1.0">
    <head>
        <title>
            Interact With Me
        </title>
        <link rel="stylesheet" href="wfstyle.css">
    </head>
    <body>
        <header class = "black">
            <h1>Poetry Flows</h1>
        </header>
        <nav class = "navigation">
            <a href = "./"><span class = "fill">Home</span></a>
            <a href = "./more/">More Poetry</a>
        </nav>
        <div class = "container" id = "container">
            <div class = "row" id = "golden">
                <a href = "./golden/">
                    Golden Hour                 
                </a>
            </div>
            <div class = "row" id = "die">
                <a href = "./die/">
                    Fight
                </a>
            </div>
            <div class="row" id = "hanahaki">
                <a href = "./hanahaki/">
                    how could something so beautiful be so cruel
                </a>
            </div>
            <div class = "row" id = "more">
                <a href = "./more/">
                    Read More
                </a>
            </div>
        </div>
        <footer>
            <h4>
                Michelle Yang 2020
            </h4>
        </footer>

    </body>
</html>