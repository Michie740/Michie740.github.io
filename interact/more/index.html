<!DOCTYPE html>
<html>
    <meta charset="utf-8">
    <meta name = "viewport" content = "width=device-width, initial-scale=1.0">
    <head>
        <title>
            More Poems
        </title>
        <link rel="stylesheet" href="../wfstyle.css">
        <link href="data:image/x-icon;base64,AAABAAEAEBAQAAEABAAoAQAAFgAAACgAAAAQAAAAIAAAAAEABAAAAAAAgAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAcMbmAAsbIQB20vUAgN3/AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABEREREREQAAMxERETETMAADMyMzIzIwAAMyMiMyMzMAAERERCRERAAAREREREJEAABEQkJCJEQAAEREJEREJAAARCRERCREAARCQiQkQkQABEREREJERAAEREREREREAERDNERDREAAMzMzMzMzAAAAAAAAAAAAD//wAAgAcAAIADAADAAwAAwAEAAOABAADgAQAA4AEAAOABAADgAQAAwAEAAMABAADAAQAAgAMAAIAHAAD//wAA" rel="icon" type="image/x-icon" />
    </head>
     
    <body class = "morepage">
        <header id = "more">
            <h1>More Poems</h1>
        </header>
        <nav class = "navigation">
            <a href = "../">Home</a>
            <a href = "../more/">More Poetry</a>
        </nav>
        <div class = "poem">
            <nav id = "nav2">
                <a href = "#animal"><u>1</u></a>|
                <a href = "#fridge"><u>2</u></a>|
                <a href = "#quatrains"><u>3</u></a>|
                <a href = "#crush"><u>4</u></a>|
                <a href = "#shakespeare"><u>5</u></a>
            </nav>
            <br>
            <div class = "content">
                <div id = "animal">
                    <br><br><br>

                    <b> A Ferocious Battle </b> 
                    <br><br>
                    Braver than flying rats <br>
                    More cowardly than pizza rats <br>
                    It circles the blanket as <br>
                    light falls through the still-wet leaves  <br>
                    of the still-wet trees <br>
                    surrounding us. <br>
                    (like a spotlight on a ring) <br>
                    (like we are fighters sizing each other <br>
                    up) <br>
                    (like a sheriff and a bandit in the Wild West) <br>
                    <br>
                    Do I throw something? <br>
                    one arm raises, ready to attack <br>
                    Do I defend? <br>
                    its tail rustles, eyes dont <br>
                    break contact <br>
                    Do I move? <br>
                    one ear twitches, waiting for one mistake <br>
                    <br>
                    I put down my food.<br>
                    eyes narrow<br>
                    Tension swells.<br>
                    inches closer<br>
                    I raise my umbrella.<br>
                    pause<br>
                    The umbrella e x p a n d s.<br>
                    silence<br>
                    .<br>
                    I lower it.<br>
                    It’s gone.<br>
                    Tree rats will not win today.<br>
                </div>
                <br>

                <div id = "fridge">
                    <br><br><br>
                    <hr>
                    <br>
                    <b> Broken Willpower </b> 
                    <br><br>
                    A looming presence with an icy inside <br>
                    holding what keeps me alive. <br>
                    <br>
                    Mostly empty it stays <br>
                    in this foreign space <br>
                    always just one door away.<br>
                    <br>
                    It has one job, and does it very well<br>
                    to hold all its contents in its freezing prison cell.<br>
                    <br>
                    ‘Free the prisoners!’ It calls, day after day,<br>
                    when I’m hungry, or I’m bored, or I’m making mistakes.<br>
                    <br>
                    And mistakes I make, when my willpower breaks,<br>
                    And I open the door to greet my gluttonous fate.<br>
                </div>

                <div id = "quatrains">
                    <br><br><br>
                    <hr>
                    <br>
                    <b> Eden </b> 
                    <br><br>
                    Take a look around, what do you see?<br>
                    Coffee, Croissants, Waffles, and Tea<br>
                    Instagram foodies and people on dates,<br>
                    Students with laptops and workers who wait.<br>
                    <br>
                    Smooth sounds flow through the open air<br>
                    following the aroma of espresso, always there.<br>
                    There would be a meet cute, if this were a show<br>
                    It’s nice to think about - it won’t happen though.<br>
                    <br>
                    String lights on wooden walls and art in ceramic cups.<br>
                    Just what about it is there, to be so fond of?<br>
                    Is it the overpriced food or the immaculate vibes?<br>
                    Or the breath-taking aesthetics that pleases the eye?<br>
                    <br>
                    What is it that I’m so fond of, just what?<br>
                    I sit down and waves of nostalgia fill me up -<br>
                    Of sweet moments and fun and laughs without reason<br>
                    and memories of friends and college app season.<br>
                    <br>
                    And that’s when it hits me, that mysterious reason,<br>
                    it once was my own personal Eden.<br>
                    Sit here with me and then maybe you’ll see,<br>
                    there’s more to this place than just coffee and tea.<br>
                    
                </div>
                <div id = "crush">
                    <br><br><br>
                    <hr>
                    <br>
                    <b> crush >///< </b> 
                    <br><br>
                    Long conversations from day until night<br>
                        it’s<br>
                        so easy<br>
                    to talk about everything anything nothing<br>
                    A budding sensation sneaks up<br>
                    <br>
                    and makes the world be a little brighter<br>
                        it’s<br>
                        so pretty<br>
                    to dream of yesterday reminisce tomorrow<br>
                    A stroke of serendipity <br>
                    <br>
                    maybe that created happier months<br>
                        it’s<br>
                        so welcome<br>
                    to be comfortable in silence with rosy cheeks <br>
                    And a rosier heart inside<br>
                    <br>
                    which speeds up and beats a little quicker<br>
                        it’s<br>
                        so thrilling<br>
                    to daydream about worlds of possibilities<br>
                    But for now, this much is enough<br>
    
                </div>
                <div id = "shakespeare">
                    <br><br><br>
                    <hr>
                    <br>
                    <b> in this prison built with love </b> 
                    <br><br>
                    In a forest shrouded with mystery <br>
                    where a wake of creatures have come to weep <br>
                    lay a girl with ethereal beauty<br>
                    calm and serene, cursed to eternal sleep<br>
                    <br>
                    A curse from her own naive desire<br>
                    when the sin of temptation grew too much<br>
                    the blood red apple she had to acquire <br>
                    and take a bite of soon as it was touched <br>
    
                    In a glass coffin she lays, still breathing<br>
                    unbeknownst to the sorrow in the air<br>
                    they lament: no more clandestine meetings<br>
                    but still, she's here, even now, the most fair <br>
                    <br>
                    And here she waits, in this prison built with love<br>
                    cold and unfeeling, like the stars above <br>
                    <br>
                    <hr>
    
                    <a href = "#more"> ^ Back to Top ^</a>
    
                </div>
            </div>
            
            <br>
        </div>
        <footer>
            <h4>
                Michelle Yang 2020
            </h4>
        </footer>
        <script>
            window.onscroll = function() {myFunction()};
            
            var navbar = document.getElementById("nav2");
            var sticky = navbar.offsetTop;
            
            function myFunction() {
              if (window.pageYOffset >= sticky) {
                navbar.classList.add("sticky")
              } else {
                navbar.classList.remove("sticky");
              }
            }
            </script>
    </body>
</html>