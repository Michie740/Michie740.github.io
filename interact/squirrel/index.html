<!DOCTYPE html>
<html>
    <meta charset="utf-8">
    <meta name = "viewport" content = "width=device-width, initial-scale=1.0">
    <head>
        <title>
            Battle
        </title>
        <link rel="stylesheet" href="assets/squirrelstyle.css">
    </head>
    <body>
        <header>
            <div class = "firstwords">
                The Battle
            </div>
        </header>
            <div class = "links">
                <a href = "../">
                    Back
                </a>
                <a href = "one/">
                    Next
                </a>

            </div>

        <footer>
            Michelle Yang 2020
        </footer>
    </body>
</html>