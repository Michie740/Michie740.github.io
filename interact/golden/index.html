<!DOCTYPE html>
<html>
    <meta charset="utf-8">
    <meta name = "viewport" content = "width=device-width, initial-scale=1.0">
    <head>
        <title>
            Golden Hour
        </title>
        <link rel="stylesheet" href="../midtermwireframe/wfstyle.css">

    </head>
     
    <body class = "goldenhrpage">
        <header>
            <h1>Golden Hour</h1>
        </header>
        <nav class = "navigation">
            <ul>
                <li>
                    <a href = "../../">Home</a>
                </li>
                <li>
                    <a href = "../midtermwireframe/wireframe.html">More Poetry</a>
                </li>
            </ul>
        </nav>
        <div class = "sun" >
            <div class = "poem" id = "sunpath">
                <div class = "sky" id = "blue">
                    The sky is blue, sun glowing over the building tops <br>
                    Countdown, less than twenty minutes on the clock <br>
                    Thirteen tons suspended over a raging roaring river <br>
                    Empty walkways, except for you and me, together <br>
                    The ground shakes, as trains rumble past <br>
                    Like an earthquake - nature's most destructive craft <br>
                </div>
                <div class = "sky" id = "bluegold">
                    Blue fades into gold, sun peeking over the building tops <br>
                    We walk side by side, wind and music to us, just props <br>
                    At the middle, at the crest, we stop - halfway <br>
                    Cameras out, ready to capture, it’s almost cliche <br>
                    Two times a day, an hour each, so easily missed <br>
                    A fleeting instance, but a time of certified bliss <br>
                </div>
                <div class = "sky" id = "gold">
                    The sky is golden, sun hidden behind the building tops <br>
                    A tranquil air settles, and just for us, time stops <br>
                    Soft streaks paint portraits across the sky <br>
                    But with you, it’s a masterpiece, and maybe that’s why <br>
                    I take a picture, capture this moment before it can pass <br>
                    Because life goes on and memories fade, but this picture, it will last. <br>        
                </div>
            </div>

        </div>
        <footer>
            <h4>
                Michelle Yang 2020
            </h4>
        </footer>
    </body>
</html>